CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 6.0.4
+
+- Version bumping related to https://github.com/alphagov/govuk_frontend_toolkit_gem/pull/15
```

Comment:
Yes I saw them, but these are related to this repo, so I thought leaving the link should highlight that this refers the gem repo. I will change it to make it a little nicer maybe.